lib/wpc/reassembly/reassembly.c

```diff
@@ -288,32 +308,43 @@ bool reassembly_get_full_message(uint32_t src_add, uint16_t packet_id, uint8_t *
 
 }
 
-void reassembly_garbage_collect(uint32_t timeout_s)
+void reassembly_garbage_collect()
 {
-    full_packet_t *fp, *tmp;
-    uint32_t messages_removed = 0;
-    HASH_ITER(hh, m_packets, fp, tmp) {
-        uint32_t last_activity =
-            (Platform_get_timestamp_ms_monotonic() - fp->timestamp_ms_epoch_last) / 1000;
-
-        /* Check if message is not getting too old */
-        if (last_activity > timeout_s)
-        {
-            LOGW("Fragmented message from src %u with id %u has no activity for more than %u s => delete it\n",
-                fp->key.src_add, fp->key.packet_id, timeout_s);
-
-            internal_fragment_t *f, *tmp;
-            LL_FOREACH_SAFE(fp->head, f, tmp) {
-                Platform_free(f->bytes, f->size);
-                LL_DELETE(fp->head, f);
-                Platform_free(f, sizeof(internal_fragment_t));
-            }
+    static unsigned long long last_gc_ts_ms = 0;
```

Comment:
Can you move it with other static of the file and prefix it with m_
(yes I know, I did it like this initially in dsap :-) )